packages/cli/src/tools/__tests__/copyFiles.test.js

```diff
@@ -47,10 +51,10 @@ test('copies text and binary files from source to destination', async () => {
 
 test('copies files from source to destination excluding directory', async () => {
   const src = path.resolve(__dirname, './__fixtures__');
+  let regexStr = path.join(src, 'extraDir');
   await copyFiles(src, DIR, {
-    exclude: [new RegExp(path.join(src, 'extraDir'))],
+    exclude: [new RegExp(RegexEscape(regexStr))],
```

Comment:
We use it as `exclude: [new RegExp(path.resolve(templatePath, 'node_modules'))]` in the source. Maybe it's not working on Windows? Could you verify on a local template and see if `node_modules` are copied? They shouldn't

Reply:
Yes they are being copied. Tried on local and remote template.

Reply:
I can't `init` using a template. Remote or local.

**For local I get this error:**
```
error Error: Cannot find module '\C:\Users\thecodrr\Source\Repos\cli\packages\cli\build\react-native-template-typescript-master-cli-v2\package.json'
error EBUSY: resource busy or locked, rmdir 'C:\Users\thecodrr\Source\Repos\cli\packages\cli\build\TestInit2'
Error: EBUSY: resource busy or locked, rmdir 'C:\Users\thecodrr\Source\Repos\cli\packages\cli\build\TestInit2'
    at Object.rmdirSync (fs.js:684:3)
    at rmdirSync (C:\Users\thecodrr\Source\Repos\cli\node_modules\fs-extra\lib\remove\rimraf.js:276:13)
    at Object.rimrafSync [as removeSync] (C:\Users\thecodrr\Source\Repos\cli\node_modules\fs-extra\lib\remove\rimraf.js:252:7)
    at Object.initialize [as func] (C:\Users\thecodrr\Source\Repos\cli\packages\cli\build\commands\init\init.js:295:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:832:11)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

For Remote:
```
debug Installing template from typescript
yarn add v1.16.0
info No lockfile found.
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
success Saved 1 new dependency.
info Direct dependencies
└─ typescript@3.5.3
info All dependencies
└─ typescript@3.5.3
Done in 7.86s.
debug Getting config from C:\Users\thecodrr\AppData\Local\Temp\rncli-init-template-jV11JR\node_modules\typescript\template.config.js
error Error: Cannot find module 'C:\Users\thecodrr\AppData\Local\Temp\rncli-init-template-jV11JR\node_modules\typescript\template.config'
error EBUSY: resource busy or locked, rmdir 'C:\Users\thecodrr\Source\Repos\cli\packages\cli\build\TestInit2'
Error: EBUSY: resource busy or locked, rmdir 'C:\Users\thecodrr\Source\Repos\cli\packages\cli\build\TestInit2'
    at Object.rmdirSync (fs.js:684:3)
    at rmdirSync (C:\Users\thecodrr\Source\Repos\cli\node_modules\fs-extra\lib\remove\rimraf.js:276:13)
    at Object.rimrafSync [as removeSync] (C:\Users\thecodrr\Source\Repos\cli\node_modules\fs-extra\lib\remove\rimraf.js:252:7)
    at Object.initialize [as func] (C:\Users\thecodrr\Source\Repos\cli\packages\cli\build\commands\init\init.js:295:26)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```
In remote, the temp directory is deleted before it can copy the template (don't know why). And in local, the path is always wrong.

Reply:
No they are not being deleted

Reply:
Tests should be fixed now.

Reply:
I'm not sure if I follow. The point of this is to exclude certain dirs, like `node_modules`. If you have a local template with `node_modules` inside, they _shouldn't_ be copied to the new project. From the code you added, it seems that this escaping needs to be done in the source, not in the test only, because this is exactly how we use it. Can you move the escape fn there?

Reply:
I did create an escape function in the source; check the latest commits.

Sure, I will rename it.

Reply:
There is no need for copying as `jest-regex-util` is already present, we can just use the function from there.